weasyl/resetpassword.py

```diff
@@ -1,6 +1,8 @@
 # resetpassword.py
 # -*- coding: utf-8 -*-
```

Comment:
``# encoding: utf-8``

Also, ``weasyl/test/test_define.py`` has the same change needed.